Wasm-container/README.md

```diff
@@ -92,43 +92,24 @@ $ make PACKAGE=vertica.deb VERTICA_VERSION=11.0.0-0
 
 # Using the container
 
-## Setting up your Web Assembly toolbox
+## Setting up your Web Assembly environment and toolbox
 
-To use this installation, before you run the container interactively, have it run the shell script `/usr/WebAssembly/tools/copy-template-to-sandbox` with an argument specifying the location of your WebAssembly toolbox (referred to as `$WASMHOME` in the commands illustrated below).
+### `setup-toolbox`
 
-The following set of commands illustrate how one might set up one's Wasm environment and then invoke the `/usr/WebAssembly/tools/copy-template-to-sandbox` script inside the container:
+To use this installation, before you run the container interactively, run set your `WASMHOME` environment variable to the name of a directory in your `$HOME` directory.  With this environment variable set, run `setup-toolbox` to copy the template Rust (`.cargo`) and Wasmer (`.wasmer`) direcotries into `$WASMHOME`.  Putting these into `$WASMHOME` instead of your home directory avoids interfering with any non-wasm Rust work you might be doing.  It also creates the `$HOME/WebAssembly/.env-setup` file, which is used by `vwasm-bash` as a startup profile.
```

Comment:
stray "run" in "run set your WASMHOME..."

Reply:
thanks, fixed.